app/commands/decidim/kids/destroy_minor_account.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Kids
+    # This command destroys the minor's account.
+    # If a minor user is not confirmed (never logged), the minor user is hard destroys.
+    # If it has been confirmed, soft-destroys
+    class DestroyMinorAccount < Decidim::Command
+      def initialize(minor_user, minor_account)
+        @minor_user = minor_user
+        @minor_account = minor_account
+      end
+
+      def call
+        transaction do
+          destroy_minor_user
+          destroy_minor_data
+          destroy_minor_account
```

Comment:
```suggestion
          destroy_minor_data
          destroy_minor_account
          destroy_minor_user
```
Shoudn't we reverse the order? destroy the parent object last

Reply:
Of course. Corrected